### PR TITLE
New & updated icons

### PR DESCRIPTION
**Pull request by Donnnno**
Updated & added some icons:

Update:
- Audiotechnica Connect (sizing)
- Fujifilm Connect (removed square shape)
- NOS (removed circle shape)
- Qr Scanner (removed circle shape)
- Sudoku (removed circle shape)
- Wi-Fi automatic 
- Signal (sizing)
- Slack (new logo)
- Standard Notes (removed circle shape)
- Pocketcasts (removed circle shape)
- Osmand (added thiccness)
- Starkeverben (removed circle shape)
- Action Dash (sizing)
- Action Launcher (sizing)
- Solitaire (removed circle shape)
- Fake traveler (updated icon)
- Discord (the old icon didn't fit and it's now matches the squircle shape of this icon pack)

New:
- Creative Cloud
- Element (formerly riot.im)
- Happening
- Posidon Launcher
- ACM
- Pastebin
- Phone Manager
- Moon Reader Pro
- DSfile
- Cyberghost
- Adobe Scan
- SD Maid
- Calculator++
- Collabora Office
- Docus
- LTE Cleaner
- PixelLab
- Pegasus
- Whatsapp web to go
- Kamusku
- Quran
- Tracker control
- Finanzassistent Postbank
- Aard 2
- AIMP
- Alarmio
- APK Kit
- Explorer
- FFUpdater
- Firefox Preview
- OpenVPN Connect
- ReadEra
- Root Explorer
- SAI
- SD Maid
- Wifi Key View
- Everdo

Appfilter:
- Aurora Droid x2
- Moved Adobe lines together
- Changed RiotX to Element (new name & icon)
- Mega 
- Windscribe
- Firefox Nightly
- ab - Music Player GO
- Gallery
- UntrackMe Lite
- Simple SMS Messenger
- HMD global camera
- NOS
- Nextcloud Notes

These issues can be closed: 

- #28
- #74
- #254
- #263
- #306
- #314 
- #316
- #318 
- #319 
- #338 
- #342 
- #345

### CODE REVIEW

**Comment by dkanada** [comment]
> added thiccness

:laughing:

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -2253,21 +2345,24 @@
     <item component="ComponentInfo{im.vector.alpha/im.vector.activity.LoginActivity}" drawable="riot" />
     <item component="ComponentInfo{im.vector.alpha/im.vector.activity.SplashActivity}" drawable="riot" />
 
-    <!-- RiotX -->
-    <item component="ComponentInfo{im.vector.riotx/im.vector.riotx.features.Alias}" drawable="riot" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:2257]: I would move this to the Element block and use the new icon just so people still using the old app won't have a non-frost icon with the next update.

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -2194,6 +2285,7 @@
     <!-- ReadEra -->
     <item component="ComponentInfo{org.readera/com.readera.MainActivity}" drawable="readera" />
 
+    <item component="ComponentInfo{org.readera/com.readera.MainActivity}" drawable="readera"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:2288]: This one's on the wrong line.

File: app/src/main/res/xml/appfilter.xml [hunk 3]
```diff
@@ -3422,7 +3534,10 @@
 
     <!-- WiFiKeyShare -->
     <item component="ComponentInfo{be.brunoparmentier.wifikeyshare/be.brunoparmentier.wifikeyshare.ui.activities.WifiListActivity}" drawable="wifikeyshare" />
-
+
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:3537]: I think there are a few lines in the diff with extra spaces.

File: app/src/main/res/xml/appfilter.xml (hunk 1 above)
Comment (Donnnno) [reply app/src/main/res/xml/appfilter.xml:2257]: yes fixed this one

File: app/src/main/res/xml/appfilter.xml (hunk 2 above)
Comment (Donnnno) [reply app/src/main/res/xml/appfilter.xml:2288]: Forgot to delete that line oops.

File: app/src/main/res/xml/appfilter.xml (hunk 3 above)
Comment (Donnnno) [reply app/src/main/res/xml/appfilter.xml:3537]: I don't fully understand what's happening here. Maybe I removed a space and added another or something like that.

**Pull request merged**